lib/restbase.js

```diff
@@ -268,7 +279,7 @@ RESTBase.prototype._request = function (req) {
         childReq.params = match.params;
 
         // Call the handler with P.try to catch synchronous exceptions.
-        return P.try(handler, [childRESTBase, childReq])
+        var request = P.try(handler, [childRESTBase, childReq])
```

Comment:
`request` is confusing as a name since we usually use `req` / `request` to denote the actual request object, but here it's a Promise. Maybe `reqHandlerPromise` or `reqHandling` would be more appropriate.